packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -277,7 +277,9 @@ export class GutenbergEditorPage {
 		await this.searchBlockInserter( blockName );
 		await frame.click( `${ selectors.blockInserterResultItem } span:text("${ blockName }")` );
 		// Confirm the block has been added to the editor body.
-		return await frame.waitForSelector( `${ blockEditorSelector }.is-selected` );
+		return await frame.waitForSelector( `${ blockEditorSelector }.is-selected`, {
+			state: 'attached',
```

Comment:
Hmm, can you explain this one to me? I'm not sure I quite follow how it fixes #59929. 

It looks like the problem was that we were interacting with a hidden element. `watiForSelector` defaults to checking for `visible` state, which should be more restrictive that `attached`, so we're actually moving to be less restrictive here. Is that what we need?

Alternatively, would it make sense to return a `locator` object here? That might help if the problem is that elements keep changing state after the fact!

Reply:
I don't recall which situation called for this change - I seem to recall a situation with certain blocks that are more complex (eg. constituting multiple smaller blocks) could encounter issues where the overall parent block would be present in the DOM, but not actually 'visible'. What was visible instead were the child blocks that form the parent block.

With that said though, it looks like the permafails in WPCOM Gutenberg mobile/desktop builds that prompted this PR in the first place is no longer occurring. Some changes contained in this PR are worth keeping, but this specific change should no longer be necessary and so my plan is to remove this change from this PR.